RealmSwift/Util.swift

```diff
@@ -47,6 +47,110 @@ internal func gsub(pattern: String, template: String, string: String, error: NSE
                                            withTemplate: template)
 }
 
+// MARK: ObjectiveCBridgeable
+
+// Used for conversion from Objective-C types to Swift types
+internal protocol ObjectiveCBridgeable  {
+    /* FIXME: Remove protocol once SR-2393 bridges all integer types to `NSNumber`
```

Comment:
Doesn't look like the Swift core team ever intends on changing this behavior, so the fixme doesn't really apply.